### DISCUSSION TITLE
Python folding

### DISCUSSION THREAD

**Discussion by Lun4m** [LunarVim/LunarVim#3749] [Q&A] [answered]
I'm trying to use this [plugin](https://github.com/eddiebergman/nvim-treesitter-pyfold) to fold Python docstrings.
The README says I have to add these lines in the `treesiter` config:
```lua
require('nvim-treesitter.configs').setup {
    ...
    pyfold = {
        enable = true,
        custom_foldtext = true -- Sets provided foldtext on window where module is active
    }
    ...
}
```
How do I do that in Lunarvim? How do I extend the `treesiter` config?

**Comment by LostInTheLogs** [comment, marked as answer]
lvim.builtin.treesitter.pyfold = {
        enable = true
        custom_foldtext = true -- Sets provided foldtext on window where module is active
    }

**Reply by Lun4m** [reply]
This doesn't throw any error, but the folding doesn't work. Maybe it's an issue with the plugin or with my config. :sweat_smile: 
Thanks anyway for the answer!

**Reply by LostInTheLogs** [reply]
make sure you have these set
```lua
vim.opt.foldmethod = "expr"
vim.opt.foldexpr = "nvim_treesitter#foldexpr()"
```

**Reply by Lun4m** [reply]
Yes, I have those options set, but the plugin doesn't work and I get the default treesitter folding.
I've read it might be a compatibility issue with indent-blanklines, so I'll try to investigate.
Anyway my goal was to only fold overly long docstrings, but maybe it's easier if I define things myself in my config :thinking:
